google-cloud-clients/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/admin/v2/stub/BigtableInstanceAdminStubSettings.java

```diff
@@ -516,15 +514,37 @@ public String extractNextToken(ListAppProfilesResponse payload) {
       RetrySettings settings = null;
       settings =
           RetrySettings.newBuilder()
-              .setInitialRetryDelay(Duration.ofMillis(5L))
+              .setInitialRetryDelay(Duration.ofMillis(1000L))
               .setRetryDelayMultiplier(2.0)
               .setMaxRetryDelay(Duration.ofMillis(60000L))
               .setInitialRpcTimeout(Duration.ofMillis(60000L))
               .setRpcTimeoutMultiplier(1.0)
               .setMaxRpcTimeout(Duration.ofMillis(60000L))
               .setTotalTimeout(Duration.ofMillis(600000L))
               .build();
-      definitions.put("default", settings);
+      definitions.put("idempotent_params", settings);
+      settings =
+          RetrySettings.newBuilder()
+              .setInitialRetryDelay(Duration.ofMillis(0L))
+              .setRetryDelayMultiplier(1.0)
+              .setMaxRetryDelay(Duration.ofMillis(0L))
+              .setInitialRpcTimeout(Duration.ofMillis(60000L))
```

Comment:
Rather than very long values where I have to count zeroes to see how long it is, how about Duration.ofMinutes(1) and the like?

Reply:
This code is autogenerated by gapic-generator. So it would have to change in its templates